WalletWasabi.Fluent/Controls/TagsBox.axaml.cs

```diff
@@ -132,14 +133,15 @@ protected override void OnApplyTemplate(TemplateAppliedEventArgs e)
 				return;
 			}
 
-			_autoCompleteBox.TextChanged += OnTextChanged;
-			_autoCompleteBox.DropDownClosed += OnDropDownClosed;
+			_autoCompleteBox.TextChanged += OnAutoCompleteBoxTextChanged;
+			_autoCompleteBox.DropDownClosed += OnAutoCompleteBoxDropDownClosed;
+			_autoCompleteBox.TemplateApplied += OnAutoCompleteBoxTemplateApplied;
```

Comment:
unsubscribe this?

shouldnt you just call _autoCompleteBox.ApplyTemplate() and not wait for it to happen on its own.

if template is already applied... nothing will happen.